infra/modules/database/dms-role.tf

```diff
@@ -6,8 +6,9 @@ resource "aws_iam_policy" "dms_access" {
   name   = "dms-access"
   policy = data.aws_iam_policy_document.dms_access.json
 }
+
 resource "aws_iam_role" "dms_access" {
-  name               = "dms-access-role"
+  name_prefix        = "dms-access-role"
```

Comment:
This change is to ensure the DMS can deploy to dev / staging / prod. Roles must have unique names